scss/_patterns_equal-height-row.scss

```diff
@@ -0,0 +1,147 @@
+@import 'settings';
+
+@mixin vf-p-equal-height-row {
+  .p-equal-height-row {
+    display: grid;
+    grid-template-columns: subgrid;
+    position: relative;
+
+    .p-equal-height-row__col {
+      border-top-color: $colors--theme--border-default;
+      border-top-style: solid;
+      border-top-width: 1px;
+      display: grid;
+      grid-column: span $grid-columns-small;
+      grid-row: span 4;
+      grid-template-rows: subgrid;
+      position: relative;
+
+      .p-equal-height-row__col-item {
+        grid-column: span $grid-columns-small;
+      }
+
+      .p-equal-height-row__col-item:first-child {
+        grid-row: auto;
+      }
+
+      @media screen and ($breakpoint-small <= width < $breakpoint-large) {
+        grid-column: span $grid-columns-medium;
+        grid-template-columns: subgrid;
+        margin-top: -1px;
+
+        .p-equal-height-row__col-item {
```

Comment:
It's fine to call this `p-equal-height-row__item` - we don't have to explicitly keep the whole structure in the naming.